tests/automation_framework/src/libs/test_base.py

```diff
@@ -14,14 +14,12 @@ def __init__(self):
         self.uri_client = env.uri_client
         self.build_request_output = {}
 
-    def setup_and_build_request_register(self, input_file):
-        pre_test_output = pre_test_worker_env(input_file)
+    def setup_and_build_request_worker_register(self, input_file):
         request_obj, action_obj = build_request_obj(input_file)
         self.build_request_output.update({'request_obj': request_obj})
         return 0
 
-    def setup_and_build_request_lookup(self, input_file):
-        pre_test_output = pre_test_worker_env(input_file)
```

Comment:
Removed the call to pre_test_worker_env as for worker_lookup nothing needs to be done.